app/components/contributor_signal_settings/contributor_signal_settings.rb

```diff
@@ -11,5 +11,25 @@ def initialize(contributor:, **)
     private
 
     attr_reader :contributor
+
+    def completed_onboarding_text
+      if contributor.signal_phone_number.present?
+        t('.complete.text.phone_number',
+          name: contributor.name,
+          phone_number: contributor.signal_phone_number&.phony_formatted)
+      else
+
+        t('.complete.text.uuid',
+          name: contributor.name,
+          uuid: contributor.signal_uuid)
+      end
```

Comment:
This is maybe weird @roschaefer ... I'm not sure that displaying the UUID for the clients is interesting for them